extensions/wikia/AdEngine/js/lookup/rubicon/rubiconFastlane.js

```diff
@@ -212,6 +212,18 @@ define('ext.wikia.adEngine.lookup.rubicon.rubiconFastlane', [
 				parameters[params.key] = params.values;
 			}
 		});
+
+		if (
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the nested one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ACollapsibleIfStatements)